doc/changelog/03-notations/11240-rew-dependent.rst

```diff
@@ -0,0 +1,5 @@
+- **Added**
+  Added :g:`rew dependent` notations for the dependent version of
+  :g:`rew` in :g:`Coq.Init.Logic.EqNotations` to allow pretty-printing
+  and pretty-parsing for :g:`match` statements on :g:`Logic.eq`
```

Comment:
What is "pretty-parsing"?

Reply:
I'm using it to mean the reverse of "pretty-printing" --- being able to parse prettier versions of things.

Reply:
Oh.  Isn't pretty-printing and "pretty-parsing" the purpose of notations?  So a bit of a tautological statement here?  Maybe just say "... to improve the display and parsing of :g:\`match` statements"?

Reply:
Done